kubernetes/config/authn_plugin/authn_plugin.c

```diff
@@ -51,10 +54,13 @@ authn_plugin_t *create_authn_plugin(const char *name)
 
     return plugin;
 
-  error:
+    error:
```

Comment:
Could you please keep the original indentation because it is formatted by our tool ?

Reply:
Fixed